Prob2/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿public class Solution
```

Comment:
@noor848 it is better to keep the class name the same as the file name (and default name) Program, and just a suggestion, try to isolate the code so you can move the method of solution into a split class